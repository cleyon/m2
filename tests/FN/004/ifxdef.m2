@define A 1
A is @ifx{defined(A)}{defined}{not defined}@.
B is @ifx{defined(B)}{defined}{not defined}@.
