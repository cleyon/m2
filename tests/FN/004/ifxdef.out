A is defined.
B is not defined.
